README.md

```diff
@@ -1,62 +1,74 @@
-# Secure Javascript Sandbox
+# Sandboxed Javascript Environment
 
-This is an experimental library to demonstrate that it is possible to use membranes to secure the object graph of a javascript environment without introducing identity discontinuity.
+This is an experimental library to demonstrate that it is possible to use membranes to create an object graph of a javascript environment without introducing identity discontinuity.
 
 ## Goals
 
-* The secure environment must have its own set of intrinsics.
-* Code executed inside the secure environment cannot observe the sandbox.
-* Mutations on the object graph should only affect the secure environment.
+* The sandboxed environment must have its own set of intrinsics.
+* Code executed inside the sandboxed environment cannot observe the sandbox.
+* Mutations on the object graph should only affect the sandboxed environment.
 
 ## Non-goals
 
-* Poisoning is still possible via the membrane by providing object-likes through the membrane that could be used by the outer realm to perform an operation that leaks primitive values that are relevant.
+* Argument poisoning is still possible via the membrane by providing object-likes through the membrane that could be used by the outer realm to perform an operation that leaks primitive values that are relevant.
 * This library does not provide security guarantees, those must be implemented on top of the distortion mechanism.
 
+## Terminology
+
+In order to make it easier to explain how this library works, we use a color code to identify objects and values in general from both sides of the sandbox:
+
+* Blue Realm is the Javascript Realm that is not sandboxed.
```

Comment:
JavaScript is written as Javascript and javascript. I'm ok with whatever as long as it's consistent 🙃

Reply:
lets go with the formal one: JavaScript